dist/assets/b7540080.app.min.js

```diff
@@ -0,0 +1 @@
=b,this.frameHeight=c,this.framesCount=d,this.element=e,this.initializeStyles(),this.initializeFrames(),this.initializeStripImages()}return a.prototype.initializeStyles=function(){return this.element.style.position="absolute"},a.prototype.initializeFrames=function(){var a,b,c;for(this.frames=[],a=b=0,c=this.framesCount-1;c>=0?c>=b:b>=c;a=c>=0?++b:--b)this.frames.push(new d(0,this.frameHeight*a));return this.frameIndex=0},a.prototype.initializeStripImages=function(){return this.stripImageElement=this.element.getElementsByTagName("img")[0],this.stripImageElement.style.width=""+this.frameWidth+"px",this.stripImageElement.style.height=""+this.frameHeight*this.framesCount+"px"},a.prototype.currentFrame=function(){return this.frames[this.frameIndex]},a.prototype.moveFrameToNext=function(){return this.frameIndex++},a.prototype.isAtLastFrame=function(){return this.frameIndex===this.framesCount-1},a.prototype.stripImageDidLoad=function(){},a.prototype.loadStripImage=function(){return this.stripImageElement.addEventListener("load",this.stripImageDidLoad,!1),this.stripImageElement.src=this.image_uri},a.prototype.activate=function(){return this.element.classList.remove("hide")},a.prototype.deactivate=function(){return this.element.classList.add("hide")},a.prototype.moveToCurrentFrame=function(){return this.currentFrame()||this.initializeFrames(),this.element.style.top=""+-1*this.currentFrame().topPosition+"px",this.element.style.left=""+-1*this.currentFrame().leftPosition+"px"},a.prototype.resetFrame=function(){return this.frameIndex=0,this.moveToCurrentFrame()},a}(),b["default"]=c}),define("multiSceneMovie/vastRequest",["exports"],function(a){"use strict";var b,c=function(a,b){return function(){return a.apply(b,arguments)}};b=function(){function a(a,b,d){this.url=a,this.eventEmitter=b,this.callback=d,this.setTrackers=c(this.setTrackers,this),this.clickThrough=void 0,this.url&&this.run()}return a.prototype.valid=function(){return this.url?!0:!1},a.prototype.run=function(){return this.requestHandler(this.setTrackers)},a.prototype.setTrackers=function(a){var b,c,d,e,f;if(a){for(c=[],Array.prototype.forEach.call(a.querySelectorAll("Linear VideoClicks ClickThrough"),function(a){return c.push({event:"clickThrough",url:a.textContent,type:"img"})}),Array.prototype.forEach.call(a.querySelectorAll("Linear TrackingEvents Tracking"),function(a){return c.push({event:a.getAttribute("event"),url:a.textContent,type:"img"})}),Array.prototype.forEach.call(a.querySelectorAll("Impression"),function(a){return c.push({event:"start",url:a.textContent,type:"img"})}),f=[],d=0,e=c.length;e>d;d++)b=c[d],f.push(this.callback(this.eventEmitter,b));return f}},a.prototype.requestHandler=function(a){var b;this.xhr||(this.xhr=new XMLHttpRequest),this.xhr.addEventListener("load",function(b){return function(){return a(b.xhr.responseXML),b.xhr.removeEventListener("load",arguments,!1)}}(this),!1);try{return this.xhr.open("GET",this.url),this.xhr.send(null)}catch(c){b=c}},a}(),a["default"]=b});
```

Comment:
Line is too long.<br>'a' is already defined.<br>Possible strict violation.<br>Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.<br>Wrap an immediate function invocation in parens to assist the reader in understanding that the expression is the result of a function, and not the function itself.<br>A constructor name should start with an uppercase letter.<br>Expected '{' and instead saw 'throw'.<br>Expected '{' and instead saw 'return'.<br>Too many errors. (100% scanned).